src/lasp_plumtree_broadcast_distribution_backend.erl

```diff
@@ -1005,11 +1005,11 @@ do(Function, Args) ->
 -endif.
 
 %% @private
-log_transmission(Term) ->
+log_transmission(Type, Payload, PeerCount) ->
     try
         case lasp_config:get(instrumentation, false) of
             true ->
-                ok = lasp_transmission_instrumentation:log(server, Term, node()),
+                ok = lasp_transmission_instrumentation:log(server, Type, Payload, PeerCount, node()),
```

Comment:
Again, I'm not sure what ```server``` means here, as this function can be triggered by any node in the system -- one of which will be acting as a client and server both.